lib/jxl/enc_modular.cc

```diff
@@ -1584,7 +1584,6 @@ void ModularFrameEncoder::AddACMetadata(size_t group_index, bool jpeg_transcode,
     }
   }
   image.channel[2].w = num;
-  image.channel[2].resize();
```

Comment:
This is the one spot where the values are already in the plane and need to be kept — just keeping the larger plane and changing its width also does the job (doesn't release the unused acs/qf values, but that likely doesn't really make any difference for encode memory usage).